includes/class-sensei-modules.php

```diff
@@ -205,34 +207,57 @@ public function get_lesson_module_if_exists( $post ) {
 		return $lesson_module;
 	}
 
-	private function render_module_select_for_course( $lesson_course, $lesson_module = 0 ) {
-		// Get the available modules for this lesson's course
-		$modules = $this->get_course_modules( $lesson_course );
+	/**
+	 * Renders the lesson module select input.
+	 *
+	 * @param int|null $course_id
+	 * @param int|null $current_module_id
```

Comment:
The parameters and return type are missing the description. I often refer to the [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods) when I'm unsure about commenting something or as a refresher.

Reply:
Thanks for the tip. Addressed in https://github.com/Automattic/sensei/pull/4421/commits/e46bc9d2cef52b8596bcbf2c744432bd724f349c.